elife/docker-scripts/docker-prune

```diff
@@ -1,10 +1,15 @@
 #!/bin/bash
 set -e
 
-docker container prune
-docker volume prune
+docker container prune -f
+docker network prune -f
+docker volume prune -f
 
-#docker images prune
 echo "Clean up all images not used in the last 14 days"
+limit=$(date +%Y-%m-%d -d "14 days ago")
+docker image prune -a --filter "until=$limit" -f
+
 # https://github.com/moby/moby/issues/9054#issuecomment-227920187
-docker images | grep -v REPOSITORY | awk '{print $3}' | xargs -n 1 docker history | grep -v missing | grep -v "^IMAGE" | egrep -v "day|hour|minute|second" | awk '{print $1}' | sort | uniq | xargs -n 1 docker inspect | jq -r .[0].RepoTags[] | xargs docker rmi
+#images="/tmp/$(date +%Y-%m-%dT%H:%M:%S).txt"
+#docker images | grep -v REPOSITORY | awk '{print $3}' | xargs -n 1 docker history | grep -v missing | grep -v "^IMAGE" | egrep -v "day|hour|minute|second" | awk '{print $1}' | sort | uniq > "$images"
+#cat "$images" | xargs -n 1 docker inspect | jq -r .[0].RepoTags[] | xargs docker rmi
```

Comment:
Can remove all these comments now?

Reply:
They are in a commit now (some new code I tried out) so yes